views/patient_authorization.html

```diff
@@ -53,7 +53,7 @@ <h3>
           <div id="consent_patients">
               {% for patient in patients %}
                 <div class="list_item">
-                  <input type="radio" id="{{ patient.patient_id }}" name="patient" value="{{ patient.patient_id }}">
+                  <input type="checkbox" id="{{ patient.patient_id }}" name="patient" value="{{ patient.patient_id }}">
```

Comment:
For launch/patient, it's only valid to have one entry checked.

Reply:
ah. I've only been selecting one for launch/patient. But we re-use this screen here for multi-patient selection too. Maybe I should create a create conditional rendering for single / multipatient launch.